internal/k8s/controller.go

```diff
@@ -1603,6 +1603,11 @@ func (c *AviController) Start(stopCh <-chan struct{}) {
 		informersList = append(informersList, lib.AKOControlConfig().AdvL4Informers().GatewayInformer.Informer().HasSynced)
 		go lib.AKOControlConfig().CRDInformers().AviInfraSettingInformer.Informer().Run(stopCh)
 		informersList = append(informersList, lib.AKOControlConfig().CRDInformers().AviInfraSettingInformer.Informer().HasSynced)
+		// Added here to ensure L4RuleInformer runs in AdvancedL4 Test cases.
+		if lib.AKOControlConfig().L4RuleEnabled() {
+			go lib.AKOControlConfig().CRDInformers().L4RuleInformer.Informer().Run(stopCh)
+			informersList = append(informersList, lib.AKOControlConfig().CRDInformers().L4RuleInformer.Informer().HasSynced)
+		}
```

Comment:
Do we need to add the healthmonitor informer here as well?

Reply:
Done.